explorer/testdata/alias/member_name_alias.carbon

```diff
@@ -39,43 +39,43 @@ alias i32IM = i32.(IM);
 fn Main() -> i32 {
   var a: A = {.n = 0};
   // TODO: Use != once we support it.
-  if (not (A.(IF)() == 1)) {
+  if (not(A.(IF)() == 1)) {
```

Comment:
I'd expect this to be `not (condition)` because `not condition` will often work (no parentheses). Maybe `not` should be set as a keyword?

Reply:
I knew I'd seen it..

![image](https://user-images.githubusercontent.com/16046346/192883172-5b8d79db-2da9-4f69-9d22-b1dce9009f7a.png)

So its likely really about Carbon team choosing your defaults

Reply:
In Google C++ style, it's always `&&`, `||`, and `!`. As a consequence, any clang-format behavior for Google C++ style around the keyword equivalents `and`, `or`, and `not` will be effectively unused.

For Carbon, it should probably just always have a space after `not`, suppressing the C++ `!` behaviors.

Reply:
Though probably fine to just set SpaceAfterLogicalNot as default if that fixes things.